Photon-Tinker/Mesh/MeshSetupDevice.swift

```diff
@@ -193,26 +193,4 @@ extension ParticleDeviceType : CustomStringConvertible {
 
         return isSNPrefix(prefix: "b31") || isSNPrefix(prefix: "p002")
     }
-
```

Comment:
+1 to this.  Better to just have a single path for this, since we always require internet access anyway.